src/coreclr/jit/hwintrinsiccodegenxarch.cpp

```diff
@@ -1977,7 +1977,7 @@ void CodeGen::genBaseIntrinsic(GenTreeHWIntrinsic* node, insOpts instOptions)
             noway_assert(simdInitTempVarNum != BAD_VAR_NUM);
 
             bool     isEBPbased;
-            unsigned offs = compiler->lvaFrameAddress(simdInitTempVarNum, &isEBPbased);
+            int offs = compiler->lvaFrameAddress(simdInitTempVarNum, &isEBPbased);
```

Comment:
I expect there is another one of these that needs to be fixed under the `_GetElement:` path